contracts/plugins/ICrunaPlugin.sol

```diff
@@ -1,13 +1,18 @@
 // SPDX-License-Identifier: CC0-1.0
 pragma solidity ^0.8.9;
 
+import {IBoundContractExtended} from "../utils/IBoundContractExtended.sol";
+import {IReference} from "../token/IReference.sol";
+import {IVault} from "../token/IVault.sol";
+import {INamed} from "../utils/INamed.sol";
+
 /**
  @title ICrunaPlugin.sol
  @dev Interface for plugins
    Technically, plugins are secondary managers, pluggable in
    the primary manage, which is CrunaManager.sol.sol
```

Comment:
Couple of typos here, small but worth mentioning

Reply:
Good catch. We have to rewrite all the comments before the audit